doc: introduce a triager role (#34295)
nodejs/node: doc/guides/contributing/issues.md

including whether the behavior being seen is a bug or a feature. This discussion
is part of the process and should be kept focused, helpful, and professional.

> Objective of helping with triaging issues (in core and help repos) is to help

Reviewer: ```suggestion
The objective of helping with triaging issues (in core and help repos) is to help
```

reduce the issue backlog and keep the issue tracker healthy, while enabling

> new comers another meaningful way to get engaged and contribute.

Reviewer: ```suggestion
newcomers another meaningful way to get engaged and contribute.
```

> Any one with a reasonable understanding of the Node.js programming and the

Reviewer: ```suggestion
Anyone with a reasonable understanding of Node.js programming and the
```

project's GitHub organization plus a few contributions to the project
(commenting on issues or PRs) can apply for and become a triager. Open a PR

> on the README.md of this project with: i) requesting to be added as a triager,

Reviewer: ```suggestion
on the README.md of this project with: i) a request to be added as a triager,
```

> ii) the motivation of becoming a triager, and iii) agreement on reading,

Reviewer: ```suggestion
ii) the motivation for becoming a triager, and iii) agreement on reading,
```

> understanding and adherence to the project's [Code Of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
understanding, and adhering to the project's [Code Of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md).
````

> The triage role enables ability to carry out most common triage activities,

Reviewer: ```suggestion
The triage role enables the ability to carry out the most common triage activities,
```

> such as: apply labels, close, repoen and assign issues, among few other things.

Reviewer: ```suggestion
such as applying labels and closing/reopening/assigning issues.
```

> More details on the roles and permissions can be found [here](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization).

Reviewer: ```suggestion
For more information on the roles and permissions, see ["Permission levels for repositories owned by an organization"](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization).
```
